docs/communication.rst

```diff
@@ -7,39 +7,39 @@ CEA website
 ------------
 
 Host: squarespace.com
-Credentials: Sign-in with CEA gmail account uesm.fcl@gmail.com in `here <https://docs.google.com/document/d/19rHXlRvjB3dfZHjWvZ8GMd2tZ0aeGgznsiyd8FChGIg/edit?usp=sharing>`__.
+Credentials in `here <https://docs.google.com/document/d/19rHXlRvjB3dfZHjWvZ8GMd2tZ0aeGgznsiyd8FChGIg/edit?usp=sharing>`__.
```

Comment:
@JIMENOFONSECA I think it's great that we removed the email address - but... keeping passwords in a google document that we then post on the internet for all to find - even if there is authorization involved - seems like a security risk. We could just change it to "Ask Product Owner for access"?

Reply:
good point? maybe we can create one link in a long temr repo? maybe ETH zurich / A/S?

Reply:
I'm not really sure what a "link in a long temr repo" is... We could just specify that the Product Owner manages the passwords (maybe in a tool like [KeePass](https://keepass.info/)? but the point being that's totally up to the Product Owner) and anyone who needs access can ask him for the user/password or to create one for him... What do you think?

Reply:
I meant the WIKI of A/S, I believe that has more long-term duration than the Product Owner.

Reply:
I think the product owner is responsible for the passwords. They shouldn't be hosted anywhere. The idea being, there is always a product owner and he/she passes the collection of passwords on to the next one? Also, note that in the version of `communication.rst` above, the same document is being referenced, even though the access groups are not the same...

Reply:
What is the PO is missing? I believe at least the Scrum Master should have access to. Why not wiki? we used to store passwords there too...

Reply:
Well, the Scrum Master gets all the passwords from the PO upon taking on that role. So yes, he/she would have the passwords too. And can act as a vice password manager.

The thing about keeping them in a wiki is that it's not secure. We shouldn't really store passwords in plain text anywhere. You should have something like KeePass or LastPass etc. for storing passwords in. Yes. We used to store passwords in a wiki. That doesn't mean it's a good idea...

Reply:
got it, What about now? :-)